packages/web/components/transactions/transaction-content.tsx

```diff
@@ -48,7 +48,7 @@ export const TransactionContent = ({
   return (
     <div className="flex w-full flex-col pb-16">
       <div className="flex w-full justify-between pt-8 pb-4">
-        <h1 className="text-h3">{t("transactions.title")}</h1>
+        <h1 className="text-h3 md:text-h5">{t("transactions.title")}</h1>
```

Comment:
To me it's notable here we have an h1 tag (that gets styles applied from globals.css), with text for h3 and h5 also applied as tailwind classes (but not the fonts for h3 and h5). Thoughts?

Reply:
Yeah I've heard that from nass as well, I think it's on track. 

But, we need to find a way to also apply the font classes for h* styles as well. Perhaps as we did with `body*`, `subtitle*`, etc. classes in globals.css, we should also make classes that re-apply the text and font tailwind classes that are getting applied to the h* elements themselves.